### ISSUE TITLE
New folder for current dataUI

### ISSUE THREAD

**Issue by holbrooa** [lzim/teampsd#474]
@lzim @saveth @staceypark @dlounsbu @dlkibbe @jessfroe @TomRust @branscombj 

As I mentioned on https://github.com/lzim/teampsd/issues/468 ...

We need a folder on each facility's sharepoint page for a copy of the data UI, so I went ahead and created one. I will maintain a copy of the latest file (the one in final_datafiles from our quant team folder) in this folder for 640. Once we finally lock down the data UI and scale, I'll create a similar folder for all the other facilities, and drop a copy of the data UI in there that has fetched the clinics for that facility.

Also @lzim if you feel like changing the name of that folder or the file inside, go for it. Doesn't matter to me, and when I propagate I'll just use whatever names you put on this folder/file.

<img width="429" alt="dataUIcap" src="https://user-images.githubusercontent.com/33842369/62797175-048b7b00-baa9-11e9-9e5d-721255619410.png">

**Comment by holbrooa** [comment]
Hopefully this saves us from hitting errors that have already been fixed. In David's case, he doesn't have easy access to our shared drive folder where the master dataUI lives, so he hit an error that I had already fixed in the current version, and there wasn't a good way forward, except me or someone else fetching him another copy. So now, if you hit a bug, it's a good idea to try it with a fresh copy of the master here before submitting the bug report. 

Running all these teams in parallel has created a situation where we have a proliferation of copies of different versions of the file, and it's impossible for anyone to keep straight. It shouldn't be a problem after the data UI is finalized. For now, though, while we're still adding features and fixing bugs, hopefully this saves us from some headaches.
